composer.json

```
@@ -56,9 +58,26 @@
         "typo3/cms-tstemplate": "dev-main as 12.0",
         "typo3/cms-viewpage": "dev-main as 12.0",
         "typo3/cms-workspaces": "dev-main as 12.0",
-        "t3docs/examples": "dev-main"
+        "t3docs/examples": "dev-main",
+        "typo3/coding-standards": "^0.5.5"
```

```suggestion
        "typo3/coding-standards": "^0.7.1"
```

We should use the current version.

---

I agree and also with the Core requirements